.github/workflows/sycl_update_gpu_driver.yml

```diff
@@ -0,0 +1,49 @@
+name: Update GPU driver
+
+on:
+  schedule:
+    - cron:  '0 3 * * 2'
+
+jobs:
+  update_driver_linux:
+    runs-on: ubuntu-latest
+    if: github.repository == 'intel/llvm'
+    steps:
+    - uses: actions/checkout@v2
+    - name: Update dependencies file
+      run: |
+        version="$(devops/scripts/update_drivers.py linux)"
+        echo 'NEW_DRIVER_VERSION='$version >> $GITHUB_ENV
+    - name: Create Pull Request
+      env:
+        BRANCH: ci/update_gpu_driver
+        LLVMBOT_TOKEN: ${{ secrets.LLVM_MAIN_SYNC_BBSYCL_TOKEN }}
+      run: |
+        cd $GITHUB_WORKSPACE/src
+        git checkout -B $BRANCH
+        git add -u
+        git commit -m "[GHA] Uplift GPU RT version for Linux CI"
+        git push https://$LLVMBOT_TOKEN@github.com/${{ github.repository }} ${BRANCH}
+        gh pr create --title "[GHA] Uplift GPU RT version for Linux CI" --body "Uplift GPU RT version for Linux to $NEW_DRIVER_VERSION" --assignee @intel/dpcpp-devops-reviewers
+
+  update_driver_linux_staging:
+    runs-on: ubuntu-latest
+    if: github.repository == 'intel/llvm'
+    steps:
+    - uses: actions/checkout@v2
+    - name: Update dependencies file
+      run: |
+        version="$(devops/scripts/update_drivers.py linux_staging)"
+        echo 'NEW_DRIVER_VERSION='$version >> $GITHUB_ENV
+    - name: Create Pull Request
+      env:
+        BRANCH: ci/update_gpu_driver
+        LLVMBOT_TOKEN: ${{ secrets.LLVM_MAIN_SYNC_BBSYCL_TOKEN }}
+      run: |
+        cd $GITHUB_WORKSPACE/src
+        git checkout -B $BRANCH
+        git add -u
+        git commit -m "[GHA] Uplift GPU RT version for Nightly runs"
+        git push https://$LLVMBOT_TOKEN@github.com/${{ github.repository }} ${BRANCH}
+        gh pr create --title "[GHA] Uplift GPU RT version for Nightly runs" --body "Uplift GPU RT version for Linux Staging to $NEW_DRIVER_VERSION" --assignee @intel/dpcpp-devops-reviewers
```

Comment:
Can/should we commit this update straight to `sycl` branch?

Reply:
That's a good suggestion. I see that this workflow runs two jobs and each job creates a PR using the same source branch `ci/update_gpu_driver`. I expect one of the jobs fail or overwrite the results of the previous job.

Reply:
I'd prefer that we wait until this has been tested a cycle or two. Opening PRs as it does now, allows us to catch any mistakes in this script.

Reply:
The problem is that it not tested. "unstable" version is used for nightly testing only and I don't think it's tested by pre-commit. So, pre-commit for "unstable" driver version update will be "no-op".

Reply:
I see that @alexbatashev implemented validation of the driver update in https://github.com/intel/llvm/pull/5757, but I think it validates only the version used for pre-commit ('linux' config) and doesn't cover the version used for nightly ('linux_staging' config). Which means that pull request updating the driver version for `linux_staging` configuration won't catch regressions in pre-commit.

Reply:
But a PR will still mean we have eyes on it, rather than merging blindly.  

I will update this tomorrow to have the linux_staging side just merge directly instead of opening a PR. 

Other than that, is the rest good to go? The GPU driver has been uplifted, so once this PR is merged I'd like to manually trigger the new Action to uplift the GPU drivers on GHA.

Reply:
done